src/screens/MintInfoScreen.tsx

```diff
@@ -1,207 +1,467 @@
-                )
-                navigation.navigate('WalletNavigator', {screen: 'Wallet'})
-            } */
-            navigation.goBack()         
-        },
+interface DetailedNutInfo {
+  methods: Array<SwapMethod>;
+  disabled: boolean;
+}
+
+const iconMap: Partial<Record<keyof GetInfoResponse, IconTypes>> = {
+  'name': 'faTag',
+  'pubkey': 'faKey',
+  'version': 'faInfoCircle',
+  'contact': 'faAddressCard'
+}
+const contactIconMap: Record<string, IconTypes> = {
+  'email': 'faEnvelope',
+  'twitter': 'faTwitter',
+  'telegram': 'faTelegramPlane',
+  'discord': 'faDiscord',
+  'github': 'faGithub',
+  'reddit': 'faReddit',
+  'nostr': 'faBolt'
+}
+
+function MOTDCard(props: {info: GetInfoResponse}) {
+  const textDim = useThemeColor('textDim')
+  return (
+    <Card
+      RightComponent={
+        <View style={{justifyContent: 'center'}}>
+          <Icon icon="faCircleExclamation" color={textDim} size={20} />
+        </View>
+      }
+      headingTx="mintInfo.motd"
+      HeadingTextProps={{style: [$sizeStyles.sm, {color: textDim}]}}
+      ContentComponent={
+        <Text style={{fontStyle: 'italic'}} text={props.info.motd} />
+      }
+    />
+  )
+}
+
+function DescriptionCard(props: {info: GetInfoResponse}) {
+  const textDim = useThemeColor('textDim')
+  return (<Card
+    headingTx="mintInfo.descriptionHeading"
+    HeadingTextProps={{ style: [$sizeStyles.sm, { color: textDim }] }}
+    style={$card}
+    ContentComponent={
+      props.info && props.info.description ? (
+        <CollapsibleText
+          collapsed={true}
+          summary={props.info.description}
+          text={props.info?.description_long || ''}
+        />
+      ) : (
+        <Text
+          style={{fontStyle: 'italic'}}
+          text={translate('mintInfo.emptyValueParam', {
+            param: translate('mintInfo.descriptionHeading'),
+          })}
+        />
+      )
+    }
+  />)
+}
+
+function NutItem(props: {
+  enabled: boolean,
+  nutNumber: string,
+  display: 'row' | 'small',
+  width?: DimensionValue
+  nutInfo?: DetailedNutInfo
+}) {
+  const textDim = useThemeColor('textDim')
+  const $nutIcon: ViewStyle = { paddingHorizontal: 0 }
+
+  return (
+    <View style={[$nutItem, { width: props.display === 'row' ? '100%' : (props?.width ?? 'auto')}]}>
+      <Icon
+          icon={props.enabled ? 'faCheckCircle' : 'faXmark'}
+          color={props.enabled ? colors.palette.success200 : colors.palette.angry500}
+          size={16}
+          containerStyle={$nutIcon}
+        />
+      <Text size='xs'>NUT-{props.nutNumber}</Text>
+      {props.nutInfo && <View>
+        {props.nutInfo.methods.map((m, i) => (<View style={$nutItem} key={i}>
+          {/* <Icon icon='faBolt' color={textDim} size={16} containerStyle={$nutIcon} /> */}
+          <Text text={m.method} size='xs'/>
+          {typeof m.min_amount !== 'undefined' && typeof m.max_amount !== 'undefined' // for whatever reason, the methods don't always have min/max amounts
```

Comment:
i use explicit checks here because if i just did `{m.min_amount && ...}`, if the min_amount is 0, it would evaluate to false.